spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -257,9 +257,9 @@
       expect(next_version).to eq('1.12.0-SNAPSHOT')
     end
 
-    it 'calculates next version correctly with snapshot version' do
```

Comment:
Note that we won't support bumping `SNAPSHOT` versions automatically after this. I did this change since I think we should never find ourselves with a SNAPSHOT. This function is currently used in 2 places:
- When determining the next version automatically for automatic bumps, which uses the latest tag in the repo, which should never be a SNAPSHOT
- When calculating the next prerelease version, and the version in the release branch when we execute this should also never be a SNAPSHOT.

Do lmk if you think this is a problem and I can refine the regex to support snapshots as well.